_commands/scripting/appendOperator.md

```diff
@@ -0,0 +1,57 @@
+---
+---
+
+>\>
```

Comment:
This is not the append operator:

<img width="359" alt="screen shot 2016-03-27 at 4 49 47 pm" src="https://cloud.githubusercontent.com/assets/14142637/14067810/fda85b78-f43b-11e5-89d2-d9ad6dcac23b.png">